package.json

```
@@ -1,50 +1,31 @@
 {
-  "name": "@getluko/eslint-plugin-mobile-guidelines",
-  "author": "luko",
-  "version": "0.0.1",
-  "description": "Custom ESLint Rules & Config created by mobile team",
-  "main": "dist/",
+  "name": "eslint-getluko-react-native-guidelines",
```

```suggestion
  "name": "eslint-react-native-guidelines",
```
?

I will rename the repo once we both agree on the name